runtime/runtime-params-estimator/src/main.rs

```diff
@@ -276,7 +276,7 @@ fn main_docker(
     let project_root = project_root();
 
     let image = "rust-emu";
-    let tag = "rust-1.62.1"; //< Update this when Dockerfile changes
```

Comment:
Hm... Can't we just read this from dockerfile? Like, `std::fs::read(env!("CARGO_MANIFOST_PATH").jonin(...))`